src/consts.ts

```diff
@@ -0,0 +1,8 @@
+// TODO: Use a mapping to lookup relayer addr based on environment
+//       e.g. Record<NetworkType = 'development | testnet | mainnet', Address>
+
+// development
+export const RELAYER_ADDR = '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266';
```

Comment:
This doesn't look like a const to me - this looks like a config - can we name this file to config.ts ? (i understand we dont want to make it based on env right now and we'll do it when we need to).

Reply:
Yep